packages/app/src/cli/models/app/loader.ts

```diff
@@ -66,14 +67,20 @@ class AppLoader {
     const configuration = await this.parseConfigurationFile(AppConfigurationSchema, configurationPath)
     const extensionsPath = path.join(this.appDirectory, `${blocks.extensions.directoryName}`)
     const dotenv = await this.loadDotEnv()
-    const functions = await this.loadFunctions(extensionsPath)
-    const uiExtensions = await this.loadUIExtensions(extensionsPath)
-    const themeExtensions = await this.loadThemeExtensions(extensionsPath)
+    const {functions, usedCustomLayout: usedCustomLayoutForFunctionExtensions} = await this.loadFunctions(
+      extensionsPath,
+    )
```

Comment:
what is the custom layout? isn't always false?

Reply:
Right now it is, but it's due a change pretty soon -- we're likely to want to allow overriding the globs for the various extensions. It's better to have this field added and populated earlier, even if the value is consistent right now. There's a few bits like this where some fields are added because we have something due in the next couple of weeks.